src/geos.c

```diff
@@ -942,3 +942,37 @@ GEOSCoordSequence* coordseq_from_buffer(GEOSContextHandle_t ctx, const double* b
   }
   return coord_seq;
 }
+
+/* Copy coordinates of a GEOSCoordSequence to an array
+ *
+ * Note: this function assumes that the buffer is from a C-contiguous array,
+ * and that the dimension of the buffer is only 2D or 3D.
+ *
+ * Returns 0 on error, 1 on success.
+ */
+char coordseq_to_buffer(GEOSContextHandle_t ctx, const GEOSCoordSequence* coord_seq,
+                        double* buf, unsigned int size, unsigned int dims) {
+  char *cp1, *cp2;
```

Comment:
can move these two declaration lines (955-956) to closer to where they are needed (line 967), since they are not used under GEOS 3.10